docs/_reference/cli-docs.md

```diff
@@ -3466,8 +3466,8 @@ const perform = async (z, bundle) => {
   );
 
   // after fetching a page, set the returned cursor for the next page,
-  // or empty/null if this was the last one.
-  await z.cursor.set(response.nextPage);
+  // or an empty string if this was the last one.
```

Comment:
Small change to sync up with https://github.com/zapier/zapier-platform/pull/703/commits/d52e81defe144dd2b171328d94f7ffbaaa1f631d:
```suggestion
  // or an empty string if the cursor is null
```